src/hotspot/share/runtime/os.cpp

```diff
@@ -2105,15 +2105,19 @@ void os::pretouch_memory(void* start, void* end, size_t page_size) {
     // We're doing concurrent-safe touch and memory state has page
     // granularity, so we can touch anywhere in a page.  Touch at the
     // beginning of each page to simplify iteration.
-    char* cur = static_cast<char*>(align_down(start, page_size));
+    void* first = align_down(start, page_size);
```

Comment:
minor nit, since you are touching this, could you make it const too? (void* const)

Reply:
Touch needs a write anyway, and all related functions also do not use const here. So I would not add const for it.

Reply:
I was talking about void* const, not const void*. But nevermind, its not important.